src/cascadia/TerminalCore/TerminalDispatch.cpp

```diff
@@ -43,8 +43,8 @@ try
     RETURN_BOOL_IF_FALSE(SUCCEEDED(SizeTToShort(column, &x)) &&

```

Comment:
@j4james had mentioned in the linked issue that we should probably replace these type conversions too to use the saturating math at the same time. If these aren't coming in from `intsafe.h`, where are they coming from? We should probably either do these in a batch with this PR or ensure we do a follow up to get them too for consistency.

Reply:
Dang these aren't coming directly from `intsafe.h` but by including some win32 headers which indirectly include `intsafe.h`, these macros are still available. For example wil has it in https://github.com/microsoft/wil/blob/master/include/wil/safecast.h

This makes me feel that this PR isn't really doing much..

Reply:
Hm. Darn ok.